src/main/resources/assets/commandblockide/lang/de_de.json

```
@@ -0,0 +1,23 @@
+{
+    "arguments.editfunction.tag.unsupported": "Kann keine Funktions Tags bearbeiten",
```

```suggestion
    "arguments.editfunction.tag.unsupported": "Funktions-Tags können nicht bearbeitet werden",
```
It would make more sense this way